index.html

```diff
@@ -205,6 +245,16 @@ <h2>Searching for Learning Objects</h2>
 			</div>
 			<br><br>
 
+			<a name="Metrics"><div class="section">
+				<h2>Learning Objects Metrics</h2>
+				<hr>
+				<p>Naviage to your dashboard and select the <i>Released</i> tab. Click the 'meatball menu' (...) of the object you want to view metricses on and select the <i>info</i> option. A side panel will display showing information about your learing object. Under the <i>Metrics</i> section, you will be able to view the number of times your learning object has been downloaded and the average rating of your learning object. The average rating is composed by reviews left by CLARK users that have used your learning object. </p>
```

Comment:
I couldn't find online that Metricses is a word, metrics would be the correct word.